CHANGELOG.md

```diff
@@ -7,10 +7,17 @@
 ### Changes:
 
 -   **Breaking Changes**
+
     -   Changed `create()` and `createTemp()` to automatically set `auxCreator` to the current `this` bot.
         -   `create()` no longer takes a bot/bot ID as the first parameter. Instead, you need to use the `from()` function to set the creator ID.
         -   e.g. `create(from(bot))`.
 
+### Changes:
```

Comment:
@shaneYeti Can you change this to use the same "Changes" block that is right above?

Reply:
Should be fixed with latest push